babel.config.js

```diff
@@ -3,30 +3,28 @@
 const envConfig = {
   ignoreBrowserslistConfig: true,
   modules: 'commonjs',
-  targets: { node: true }
+  targets: { node: true },
 };
 
 if (process.env.ESM) {
-  envConfig.modules = false
+  envConfig.modules = false;
 }
 
 if (process.env.CDN) {
-  envConfig.modules = 'umd'
-  envConfig.targets = null
-  envConfig.ignoreBrowserslistConfig = false
+  envConfig.modules = 'umd';
+  envConfig.targets = null;
+  envConfig.ignoreBrowserslistConfig = false;
 }
 
 module.exports = {
   presets: [
-    [
-      require.resolve('@babel/preset-env'),
-      envConfig,
-    ],
+    [require.resolve('@babel/preset-env'), envConfig],
     require.resolve('@babel/preset-flow'),
     require.resolve('@babel/preset-react'),
   ],
   plugins: [
     require.resolve('@babel/plugin-proposal-class-properties'),
     require.resolve('@babel/plugin-syntax-dynamic-import'),
+    require.resolve('babel-plugin-macros'),
```

Comment:
we really only want this in the environments where we run jest, not prod. it doesn't hurt to have it around but it's messy to not remove it. how do we do that?

Reply:
good question, lets add it to the babel config itself like this:
```js
env: {
    test: {
      plugins: [require.resolve('babel-plugin-macros')]
    }
  },
plugins: [
  require.resolve('@babel/plugin-proposal-class-properties'),
  ...
]
```
jest should automatically set `process.env.BABEL_ENV=test`